2.js

```diff
@@ -0,0 +1,86 @@
+const readline = require('readline')
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+const isValidTile = character => {
+  if (typeof character !== 'string') return false
```

Comment:
I recommend avoiding single line control flow and return, for clarity.  As well, should this code ever require maintenance, you have deferred the work of two braces and a new line to the next developer.